packs/vsphere/actions/get_vm.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
you mean, using yaml instead of json? But I see all other actions are done with json.
Or you are talking about something else?